### PR TITLE
ajansspor.com 

### LINKED ISSUE #103408: ajansspor.com
### Issue URL (Ads)
 [https://ajansspor.com/haber/merih-demiral-yurt-disindan-teklifler-aldim-590786](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fajansspor.com%2Fhaber%2Fmerih-demiral-yurt-disindan-teklifler-aldim-590786)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.6.16
Browser: | Chrome
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Self-destructing third-party cookies (2880), <br>Block WebRTC, <br>Hide your Referrer from third-parties
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard URL Tracking, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Turkish

### PR DESCRIPTION
**Pull request by ghost**
Fixes #103408

Obsolete: `https://forum.adguard.com/index.php?threads/resolved-http-www-ajansspor-com.14193/#post-104478`

### CODE REVIEW

File: SpywareFilter/sections/specific.txt [hunk 1]
```diff
@@ -47,6 +47,7 @@
 !
 /analytics.js$domain=bataryapil.com|journey.com.tr|tozlu.com|elleshoes.com|freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|partifabrik.com|newsmax.com
 !
+||counter.ajansspor.com^
```
Comment (Sergey-Lyapin) [inline SpywareFilter/sections/specific.txt:50]: to `tracking_servers_firstparty` please

**Comment by Sergey-Lyapin** [comment]
Hi @RadicalNoise

Please add:
`ajansspor.com##.ima-ad-container`
to the turkish filter
`ajansspor.com##.detail__item-sub`
to subscriptions.txt

what about to add `ajansspor.com##.lozad` to the annoyances filter? It's the same autoplay video after each article.

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

**Comment by RadicalNoise** [comment]
Please check new ads below.

```
https://ajansspor.com/kategori/16/futbol
ajansspor.com##.big-point__status-rk

https://ajansspor.com/kategori/10/voleybol
ajansspor.com##.btcturk__banner

```

> what about to add ajansspor.com##.lozad to the annoyances filter? It's the same autoplay video after each article.

I didn't block it because it looks like a first-party TV content.

`ajansspor.com##.lozad` breaks the site. It also blocks news images.

**Comment by Sergey-Lyapin** [comment]
> I couldn't find the ima-ad-container rule in html but it was added.

I don't see it too anymore, removed the rule fot it.

**Pull request merged**